src/HtmlUtils.tsx

```diff
@@ -635,6 +638,66 @@ export function linkifyAndSanitizeHtml(dirtyHtml: string, options = linkifyMatri
     return sanitizeHtml(linkifyString(dirtyHtml, options), sanitizeHtmlParams);
 }
 
+/**
+ * Turns a given URL into an absolute one.
+ * @param {string} url The URL transform
+ * @returns {string} Absolute URL
+ */
+const getAbsoluteUrl = (() => {
+    // Use a single cached anchor element stored in a closure to avoid having to recreate
+    // a new one on every call and increase performance
```

Comment:
otherwise it sort of sounds like we missed a performance benefit 😅 
```suggestion
    // a new one on every call, which would affect performance
```